src/libs/ajax.js

```diff
@@ -1044,3 +1045,8 @@ export const Ajax = signal => {
 
 // Exposing Ajax for use by integration tests (and debugging, or whatever)
 window.Ajax = Ajax
+
+// Pulling Ajax from context allows overriding for usage outside of Terra UI.
+export const ajaxContext = createContext(Ajax)
+
+export const useAjax = () => useContext(ajaxContext)
```

Comment:
let's put jsdoc comments on these two exports and add Experimental suffix as was discussed at UIWG meeting.  A doc or PR link in each?

Reply:
I have now made all of the changes to this that I am comfortable making without further discussion. I'm happy to do follow-up PRs as needed.